configure.ac

```diff
@@ -751,6 +751,10 @@ AS_VAR_IF([PHP_VALGRIND], [no],,
       AC_DEFINE([HAVE_VALGRIND], [1],
         [Define to 1 if Valgrind is enabled and supported.])])])
 
+if test "$PHP_VALGRIND" = "yes"; then
```

Comment:
just my 2 cents, might be possible to check it only when valgrind detection is succesful (above) no ?

Reply:
Maybe. The configure.ac syntax is so confusing to me that I decided to just create a new if. :sweat_smile: If Peter prefers refactoring this, I'll go ahead.

Reply:
I totally agree. This Autoconf shell-mixed full-of-portability-concerns syntax is so horrible, that it's not something to keep for too long. Hopefully this will be replaced one day with CMake. Another syntax, but just few levels less horrible :D

Reply:
Thank you! :pray: Thank you for all your work!